ci/integration/logging/run_sg.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/env bash
+# CI script for CentOS8 job
+# purpose: spawn sg-core to process messages sent by rsyslog
+
+set -ex
+
+# enable required repo(s)
+yum install -y epel-release
```

Comment:
It would be better if we could avoid using epel-release and base this on ubi8 which is going to better align everything downstream for us. I recently migrated a couple of the other repositories to ubi8 and there should be some yum repositories loaded that will help cover things that don't exist in native ubi8.

Reply:
Yea check this out in smart-gateway repo where I use opstools.repo for qpid-proton-c-devel.

https://github.com/infrawatch/smart-gateway/blob/master/build/test-framework/run_tests.sh

All the other repos we have are on ubi8, and our base image downstream is ubi8, so I'm kind of wanting it all to be in sync as much as possible so we don't get surprised in testing or builds when importing downstream.

Reply:
Oh I was not aware that we have this dependency in opstools repo. In that case I can move to ubi8. Thanks for a pointer.